presto-tests/src/test/java/com/facebook/presto/tests/TestLocalQueries.java

```diff
@@ -58,4 +59,17 @@ public static LocalQueryRunner createLocalQueryRunner()
 
         return localQueryRunner;
     }
+
+    @Test
+    public void testCorrelatedIn()
```

Comment:
In `ATQ` there are other test cases. Can you think about other cases that should be covered?